scst-store/cert-rotation.hbs.md

```diff
@@ -0,0 +1,44 @@
+# Certificate Rotation
+
+This topic describes TLS Certificate Rotation for Supply Chain Security Tools (SCST) - Store.
+
+## Certificates
+
+By default the `use_cert_manager` setting is set to `"true"`.
+When the setting `use_cert_manager` is `"true"` the Store uses `cert-manager` to generate a CA certificate, an API certificate, and a DB Certificate.
+
+To see these certificates, run the following:
+
+```bash
+$ kubectl get certificate -n metadata-store
+NAME                    READY   SECRET                  AGE
+app-tls-ca-cert         True    app-tls-ca-cert         38d
+app-tls-cert            True    app-tls-cert            38d
+postgres-db-tls-cert    True    postgres-db-tls-cert    38d
+```
+
+Those above certificates are automatically rotated by `cert-manager`.
+The Store is able to load these certificates automatically once `cert-manager` rotates them.
+
+## Certificate duration setting
+ 
+In the `tap-values.yaml` file, `api_cert_duration`, `api_cert_renew_before`, `ca_cert_duration`, and `ca_cert_renew_before`  are configurable as shown in the following YAML:
+
+```yaml
+metadata_store:
+  ca_cert_duration: 8760h
+  ca_cert_renew_before: 1h
+  api_cert_duration: 2160h
+  api_cert_renew_before: 24h
+```
+
+Where `ca_cert_duration` is the duration that the ca certificate will be valid for. Must be given in h, m, or s. Default value is 8760h
+
+Where `ca_cert_renew_before` is the time when to renew the ca certificate before the expiration time. Must be given in h, m, or s. Default value is 1h
```

Comment:
Can you clean up this part (`the time when to renew`)? It's not quite making sense to me.